src/main/webapp/i18n/de/userSettings.json

```
@@ -27,11 +27,22 @@
                 "deleteFailure": "SSH-Schlüssel konnte nicht gelöscht werden.",
                 "deleteSuccess": "SSH-Schlüssel erfolgreich gelöscht.",
                 "cancelSavingSshKey": "Abbrechen",
-                "editExistingSshKey": "SSH-Schlüssel bearbeiten",
-                "addNewSshKey": "Neuer SSH-Schlüssel",
+                "editExistingSshKey": "Bearbeiten",
+                "viewExistingSshKey": "Ansehen",
+                "addNewSshKey": "SSH Schlüssel hinzufügen",
+                "sshKeyDetails": "SSH Schlüssel Informationen",
```

The other translations use a hyphen here:

```suggestion
                "addNewSshKey": "SSH-Schlüssel hinzufügen",
                "sshKeyDetails": "SSH-Schlüssel Informationen",
```